packages/mvsp/mvsp-alpha.en.asciidoc

```diff
@@ -19,10 +19,11 @@ h| Control
 h| Description
 
 | 1.1 Vulnerability reports
```

Comment:
Should we consider renaming this control to `External vulnerability reports` to disambiguate this?